searchcore/src/vespa/searchcore/proton/matching/matching_stats.h

```diff
@@ -82,6 +86,8 @@ struct MatchingStats
         size_t docsReRanked() const { return _docsReRanked; }
         Partition &softDoomed(bool v) { _softDoomed += v ? 1 : 0; return *this; }
         size_t softDoomed() const { return _softDoomed; }
+        Partition & doomOvertime(fastos::TimeStamp overtime) { _doomOvertime.set(overtime.sec()); return *this; }
+        fastos::TimeStamp doomOvertime() const { return fastos::TimeStamp::fromSec(_doomOvertime.max()); }
```

Comment:
why not expose the Avg here, since you want to merge it into another Avg?

Reply:
Avg is private.

Reply:
Fixed with a friend.